ui/app/app.js

```diff
@@ -190,7 +190,7 @@ App.prototype.renderAppBar = function () {
           ]),
 
           // hamburger
-          h(SandwichExpando, {
+          props.isUnlocked && h(SandwichExpando, {
```

Comment:
I kindof wish there was a way to hide these things without calling `props.isUnlocked &&` before each element.

Surely making the menu bar its own component could offer us a better abstraction.